store/displayPreferences.ts

```diff
@@ -0,0 +1,224 @@
+import { DisplayPreferencesDto } from '@jellyfin/client-axios';
+import { MutationTree, ActionTree, GetterTree } from 'vuex';
+import nuxtConfig from '~/nuxt.config';
+
+const stringToBoolean = (value: string) => value === 'True';
+const booleanToString = (value: boolean) => (value ? 'True' : 'False');
+
+export interface DisplayPreferencesState extends DisplayPreferencesDto {
+  CustomPrefs: {
+    [key: string]: string;
+  };
+}
+
+const defaultCustomPrefs: {
+  [key: string]: string;
+} = {
+  darkMode: booleanToString(
+    nuxtConfig.vuetify?.theme?.default === 'dark' || false
+  ),
+  locale: nuxtConfig.i18n?.defaultLocale || 'en'
+};
+
+const defaultState = (): DisplayPreferencesState => ({
+  CustomPrefs: defaultCustomPrefs
+});
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Can you please explain why would you need `any`?

Reply:
I just don't know the type for functions, what could I use?

Reply:
`() => void` for functions that do not return a value 

`() => unknown` for functions that we don't know what they would return

Reply:
Done